welly/quality.py

```diff
@@ -10,6 +10,227 @@
 from . import utils
 
 
+def qc_curve_group_well(well, tests, keys=None, alias=None):
+    """
+    Run tests on a cohort of curves.
+
+    Args:
+        well (welly.well.Well): Well object.
+        tests (dict): a dictionary of tests, mapping mnemonics to lists of
```

Comment:
what does test need to contain?

Reply:
Added description pointing to an example in the tests